pkgs/top-level/all-packages.nix

```
@@ -5164,6 +5164,8 @@ in
 
   pwgen = callPackage ../tools/security/pwgen { };
 
+  pwgen_secure = callPackage ../tools/security/pwgen_secure { };
```

```suggestion
  pwgen-secure = callPackage ../tools/security/pwgen_secure { };
```

Probably should rename the directory too.